src/app/containers/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from "react";
+import {
+  Button,
+  CardActions,
+  CardContent,
+  Typography,
+  Grid,
+} from "@material-ui/core";
+
+import { LogoutUser, GetNewToken } from "../../../auth/core/redux/actions";
+
+import { connect } from "react-redux";
+
+import { useDispatch } from "react-redux";
+
+import { Card, CardHeader } from "@material-ui/core";
+
+import { acceptPrivacyPolicy } from "../../../auth/core/services/privacypolicy.service";
+
+function Dashboard(props: any) {
+  const dispatch = useDispatch();
+  const [disabled, setDisabled] = useState(false);
+
+  const accessToken = localStorage.getItem("accessToken")
+    ? localStorage.getItem("accessToken")
+    : sessionStorage.getItem("accessToken");
+
+  const refreshToken = localStorage.getItem("refreshToken")
+    ? localStorage.getItem("refreshToken")
+    : sessionStorage.getItem("refreshToken");
+
+  const handleClick = () => {
+    dispatch(LogoutUser());
+  };
+
+  const getNewToken = () => {
+    if (accessToken !== null && refreshToken !== null) {
+      dispatch(GetNewToken(accessToken, refreshToken));
+    }
+  };
+
+  const userData = props.state.auth.userData ? props.state.auth.userData : null;
```

Comment:
this is same as `props.state.auth.userData?`

Reply:
Made changes.